### PR TITLE
Second part

### PR DESCRIPTION
**Pull request by AlexFalzone**
@amenuor  @tsuina

### CODE REVIEW

File: overview_on_ethereum.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Chapter on overview of Ethereum blockchain
+
+
+## Introduction:
+
+Ethereum is one of the biggest blockchains that is used in the world. The intent of it is to create alternative protocols for building decentralized applications. Is based on the *Ethereum Virtual Machine* (EVM) [[1](#references)]. The main innovation of this blockchain is the use of the *smart contract* [[2](#references)] and the use of it with the *DAPPs* (Decentralized Applications) [[3](#references)].
```
Comment (Tsuina) [inline overview_on_ethereum.md]: I would use "its purpose" instead of "The intent of it".
**It** is based

File: overview_on_ethereum.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+
+## Why we chose Ethereum:
+We can say that we chose Ethereum blockchain mainly for the following reasons:
+- Smart contracts [[2](#references)] are the main idea of the Ethereum blockchain. Nick Szabo, a computer scientist in 1994, created the term and defined it as *"A smart contract is a computerized transaction protocol that executes the terms of a contract"* [[4](#references)]. He also proposed the metaphor of smart contracts intended as vending machines. With the right inputs, a certain output is guaranteed.
```
Comment (Tsuina) [inline overview_on_ethereum.md]: main idea**s**

File: overview_on_ethereum.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+
+## Why we chose Ethereum:
+We can say that we chose Ethereum blockchain mainly for the following reasons:
+- Smart contracts [[2](#references)] are the main idea of the Ethereum blockchain. Nick Szabo, a computer scientist in 1994, created the term and defined it as *"A smart contract is a computerized transaction protocol that executes the terms of a contract"* [[4](#references)]. He also proposed the metaphor of smart contracts intended as vending machines. With the right inputs, a certain output is guaranteed. 
+An important feature is that it is permissionless, anyone can write a smart contract (and call back another smart contract inside of it) and deploy it to the network. To do this last thing is necessary a programming language. The principal are Solidity[[5](#references)] and Viper[[6](#references)].
```
Comment (Tsuina) [inline overview_on_ethereum.md]: **it** is necessary (then not sure what you mean, is it "to use a programming language" ?
last sentence: precise the principal (what?) are Solidity... principal models? programming languages?

**Review by Tsuina — COMMENTED** [review]
Hey Alex, nice I added some comments

**Comment by AlexFalzone** [comment]
I fixed the previous comments. 
@Tsuina Do you have any other comments for me?

**Review by Tsuina — COMMENTED** [review]
No other comments, thanks for fixes. I'll leave the approval to @amenuor

**Review by amenuor — APPROVED** [review]
I think we are still missing a lot of info for completing the tasks
FRA-569, FRA-568, and FRA-488

**Pull request merged**